index.html

```diff
@@ -145,24 +145,24 @@ <h1 class="title">About</h1>
 
     <section class="contact">
       <div class="headline">
-      <h2 class="title">Get Started</h2>
```

Comment:
- [OPTIONAL] It would be nice to style this element the same way it was designed in the Figma template. Your header is too bold. I would recommend reducing the `font-weight`. 👍 😎  

### Your Design

![image](https://user-images.githubusercontent.com/77106962/229895592-258e2d85-7f5c-4bd7-8462-0d26b9f75656.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/229895736-9c7fdeac-a269-4aee-9820-5c60a219f37d.png)